### DISCUSSION TITLE
StandardListView/ListView drag and drop

### DISCUSSION THREAD

**Discussion by gcavelier** [slint-ui/slint#6717] [Q&A]
Hi!

With slint, is it possible to drag and drop an item from a StandardListView (or a ListView) to another (within the same window)

I found this issue : https://github.com/slint-ui/slint/issues/1967 but it does not answer this specific question.

Thanks

**Comment by ogoffart** [comment]
There is no builtin support. But by using `item-pointer-event` callback on the StandardListView it may be possible to achieve it.
What exactly are you trying to do?

**Comment by gcavelier** [comment]
I would like to have multiple StandardListView (one for each ssh connection for example), with each StandardListView containing a tree of the files/directories on that connection (I suppose that building a TreeView should not be too difficult).

Then, I would like to move an item from one StandardListView to another to trigger a copy of that file/directory.

I tried to use `item-pointer-event` but there is no visual cue that the element is moving and I don't know how to get the destination StandardListView only with the x and y coordinates of the mouse.
